packages/transport-webrtc/test/basics.spec.ts

```diff
@@ -85,15 +85,16 @@ describe('basics', () => {
     remoteNode = await createNode()
   })
 
-  afterEach(async () => {
-    if (localNode != null) {
-      await localNode.stop()
-    }
-
-    if (remoteNode != null) {
-      await remoteNode.stop()
-    }
-  })
+  // TODO: Streams are not closing gracefully, re-introduce after https://github.com/libp2p/js-libp2p/issues/1793 is addressed
```

Comment:
The issue in #1793 does not prevent nodes from stopping.  It prevents stream data from being sent to the remote before the stream is closed.

Why does that mean these lines need to be commented out?